react/src/App.tsx

```diff
@@ -78,6 +78,22 @@ function App() {
           <sdds-breadcrumb-item current={true}>Current page</sdds-breadcrumb-item>
         </sdds-breadcrumb>
       </div>
+        <div>
+          <div className="sdds-headline-02 sdds-u-pb1">Card</div>
+          <sdds-card
```

Comment:
Should we have the native version in this example as well? :)

Reply:
I put it as a topic for today's dev check-in, I wonder the same.
If so, makes sense to have a tab on the demo page where just HTML&CSS components are.